guides/gatherers.md

```diff
@@ -66,3 +67,39 @@ facts:
 ```
 
 For extra information refer to [trento-project/agent/../gatherers/corosyncconf_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/corosyncconf_test.go)
+
+### sbd_config
+
+**Name:** `sbd_config`
```

Comment:
I would get rid off this `Name` entry. It is a duplicated value with the header.
And we shouldn't have complicated gatherer names as they are used as argument in the engine